matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/query/ReadQueries.kt

```diff
@@ -23,6 +23,9 @@ import org.matrix.android.sdk.internal.database.model.TimelineEventEntity
 import io.realm.Realm
 import io.realm.RealmConfiguration
 
+private const val MARK_OLD_EVENT_AS_READ = true
+private const val MARK_UNREAD_DUE_TO_FASTLANE = false
```

Comment:
I'm not sure declaring const for boolean returned value increases the readability of the code. Maybe a good comment is better?

Reply:
:+1: I think this is a sign that the `handleMissingEvent` is the wrong name, renaming to `hasReadMissingEvent` allows us to infer the return result is `has read` or `has not read`

Reply:
4a51583a7608e8a83add318332d0f3cbab244595